qdrant-landing/content/documentation/embeddings/premai.md

```diff
@@ -0,0 +1,477 @@
+    )
+
+    # Upload all the documents to the collection
+
+    doc_ids = list(range(len(embeddings)))
+    qdrant_client.upsert(
+        collection_name=COLLECTION_NAME, 
+        points=points
+    )
+
+    # Query your Collection 
+
+    query = "what is the extension of python document"
+    query_embedding = get_embeddings(
+        project_id=PROJECT_ID, 
+        embedding_model=EMBEDDING_MODEL, 
+        documents=query
+    )
+
+    qdrant_client.search(collection_name=COLLECTION_NAME, query_vector=query_embedding[0])
+```
+
+**JavaScript**
+
+```jsx
+import Prem from '@premai/prem-sdk';
+import { QdrantClient } from '@qdrant/js-client-rest';
+
+// Define the constants
+
+const PROJECT_ID = 123;
+const EMBEDDING_MODEL = "text-embedding-3-large";
+const COLLECTION_NAME = "prem-collection-js";
+const SERVER_URL = "http://127.0.0.1:6333"
+const DOCUMENTS = [
+    "This is a sample javascript document",
+    "We will be using qdrant and premai javascript sdk"
+];
+
+// Instantiate the clients 
+
+const premaiClient = new Prem({
+    apiKey: "xxxx-xxx-xxx"
+})
+const qdrantClient = new QdrantClient({ url: SERVER_URL });
+
+// Define helper function to create embeddings and 
+// convert it to Qdrant points 
+
+async function getEmbeddings(projectID, embeddingModel, documents) {
+    const response = await premaiClient.embeddings.create({
+        project_id: projectID,
+        model: embeddingModel,
+        input: documents
+    });
+    return response;
+}
+
+function convertToQdrantPoints(embeddings, texts) {
+    return embeddings.data.map((data, i) => {
+        return {
+            id: i,
+            vector: data.embedding,
+            payload: {
+                text: texts[i]
+            }
+        };
+    });
+}
+
+// Use the function to get embeddings and points
+let points = null
+try {
+    const embeddings = await getEmbeddings(PROJECT_ID, EMBEDDING_MODEL, DOCUMENTS)
+    points = convertToQdrantPoints(embeddings, DOCUMENTS)
+} catch (error) {
+    points = null
+    console.error('Error fetching embeddings:', error);
+}
+
+// Create a new collection to store those points and index documents
+// Comment it out once a collection is created 
+await qdrantClient.createCollection(COLLECTION_NAME, {
+    vectors: {
+        size: 3072,
+        distance: 'Cosine'
+    }
+})
+
+// Upload the documents to the collection
+if (points != null) {
+    await qdrantClient.upsert(COLLECTION_NAME, {
+        wait: true,
+        points
+    })
+    console.log("All the points are upserted to: ", COLLECTION_NAME)
+}
+
+// Query and search over the indexed documents in the collection
+const query = "what is the extension of javascript document"
+const query_embedding_response = await getEmbeddings(PROJECT_ID, EMBEDDING_MODEL, query)
+
+let response = await qdrantClient.search(COLLECTION_NAME, {
+    vector: query_embedding_response.data[0].embedding
+})
+
+console.log(response)
+```
```

Comment:
Probably remove this section. As it's redundant and inconsistent with the other embeddings pages.

Reply:
I see, okay I will remove the "Full code section"

Reply:
Done